### ISSUE TITLE
Learn about git

### ISSUE THREAD

**Issue by emmcold** [emmcold/KoochiProblems#24]
https://www.youtube.com/watch?v=4SD6rWt9wUQ

**Comment by emmcold** [comment]
@sinhaaman  can you please share some **good** materials about git that can be helpful for me.

**Comment by sinhaaman** [comment]
yes sure, give me 2 minutes. I can give a demo for 1 hour and then it could be sufficient. Meanwhile I am searching for some stuffs. It is just about practice to git. That's it, sometimes you will screw up with git, but don't worry.

**Comment by emmcold** [comment]
@sinhaaman Yesterday or the day before when you made me ignore some parts of the git, were they these folders: 
![image](https://user-images.githubusercontent.com/37145763/38770202-4bd3c6ae-400f-11e8-9744-4920ce73253c.png)

**Comment by sinhaaman** [comment]
I tried this site to learn git once. https://try.github.io/levels/1/challenges/1

This is an easy site and will teach you the basic with which you can start. Rest will follow when you start making commits in git.

Regarding previous comment, yes, I don't want those folder to be checked in. They are eclipse related folder and should not be checked into the git. Only your source code should be checked in i.e. .java files.

**Comment by sinhaaman** [comment]
@emmcold  In order to get some quiz of the git after learning. This is also a good site.
https://www.codecademy.com/learn/learn-git

I haven't tried but codeacademy is good. The above two links will make you learn git. Don't worry at all
